pkg/clustertree/cluster-manager/cluster_controller.go

```diff
@@ -307,6 +310,7 @@ func (c *ClusterController) setupControllers(
 	leafPodController := podcontrollers.LeafPodReconciler{
 		RootClient: c.Root,
 		Namespace:  "",
+		PodLock:    &SharedPodLock,
 	}
```

Comment:
The usage of the shared lock is incorrect.


The usage of the shared lock is incorrect.

Reply:
already update